db/patterns/socialhead.eno

```diff
@@ -0,0 +1,13 @@
+name: socialhead
```

Comment:
Do you think we should rename it to "Socialwidget" or "Socialhead's Socialwidget"? To distinguish it from other Socialhead offerings such as OnePixel?

Reply:
What about "Socialhead" + "Socialwidget" just like "Google" + "Analytics"?

Reply:
Yes, we can use `Socialhead Socialwidget`

Reply:
Thanks! Done.